db/migrate/20200514212601_add_public_reactions_count_columns.rb

```diff
@@ -0,0 +1,8 @@
+class AddPublicReactionsCountColumns < ActiveRecord::Migration[5.2]
```

Comment:
Isn't this the same file from https://github.com/thepracticaldev/dev.to/pull/7919 ? This should be removed.